utils/cpuload/conn.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cpuload
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/binary"
+	"os"
+	"syscall"
+)
+
+type Connection struct {
+	fd   int
+	pid  uint32
+	seq  uint32
+	addr syscall.SockaddrNetlink
+	rbuf *bufio.Reader
+}
+
+// Create and bind a new netlink socket.
+func NewConnection() (*Connection, error) {
```

Comment:
Do we want to have a separate netlink in util? It seems like we're starting to make an interface for it. I'm fine leaving here for now and making all private.